client/my-sites/people/people-list-section-header/index.jsx

```diff
@@ -17,8 +17,6 @@ import { localize } from 'i18n-calypso';
  */
 import SectionHeader from 'components/section-header';
 import Button from 'components/button';
-import ButtonGroup from 'components/button-group';
-import Tooltip from 'components/tooltip';
```

Comment:
Why the removal of all this code? I wonder why it's there?

Reply:
The old button had a tooltip and was a button group. https://github.com/Automattic/wp-calypso/pull/23085/files#diff-6e27d82feae1a8f5ae795733f6c1ea40L69